api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -8650,4 +8650,196 @@
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<tableExists tableName="order_group" />
+			</not>
+		</preConditions>
+		<comment>
+			Create order_group table
+		</comment>
+		<createTable tableName="order_group">
+			<column name="order_group_id" type="int" autoIncrement="true">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="order_set_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="patient_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="encounter_id" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="creator" type="int">
+				<constraints nullable="false"/>
+			</column>
+			<column name="date_created" type="DATETIME">
+				<constraints nullable="false"/>
+			</column>
+			<column name="voided" type="BOOLEAN" defaultValueBoolean="false">
+				<constraints nullable="false"/>
+			</column>
+			<column name="voided_by" type="int"/>
+			<column name="date_voided" type="DATETIME"/>
+			<column name="void_reason" type="varchar(255)"/>
+			<column name="changed_by" type="int"/>
+			<column name="date_changed" type="DATETIME"/>
+			<column name="uuid" type="char(38)">
+				<constraints unique="true"/>
+			</column>
+		</createTable>
+		<addForeignKeyConstraint constraintName="order_group_patient_id_fk"
+								 baseTableName="order_group" baseColumnNames="patient_id"
+								 referencedTableName="patient" referencedColumnNames="patient_id"/>
+		<addForeignKeyConstraint constraintName="order_group_encounter_id_fk"
+								 baseTableName="order_group" baseColumnNames="encounter_id"
+								 referencedTableName="encounter" referencedColumnNames="encounter_id"/>
+		<addForeignKeyConstraint constraintName="order_group_creator_fk"
+								 baseTableName="order_group" baseColumnNames="creator"
+								 referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_group_set_id_fk"
+								 baseTableName="order_group" baseColumnNames="order_set_id"
+								 referencedTableName="order_set" referencedColumnNames="order_set_id" />
+		<addForeignKeyConstraint constraintName="order_group_voided_by"
+								 baseTableName="order_group" baseColumnNames="voided_by"
+								 referencedTableName="users" referencedColumnNames="user_id" />
+		<addForeignKeyConstraint constraintName="order_group_changed_by"
+								 baseTableName="order_group" baseColumnNames="changed_by"
+								 referencedTableName="users" referencedColumnNames="user_id"/>
+	</changeSet>
+
+	<changeSet id="20151118-1640" author="bahmni" dbms="mysql">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists columnName="sort_weight" tableName="orders"/>
+				<columnExists columnName="order_group_id" tableName="orders"/>
+			</not>
+		</preConditions>
+		<comment>Adding 'order_group_id' and 'sort_weight' column to orders table</comment>
+		<addColumn tableName="orders">
```

Comment:
It's good practice to add each column in its own changeset so that precondition failure for one doesn't affect the other